templates/index.html

```diff
@@ -2,6 +2,9 @@
 
 <!-- wp:query {"tagName":"main","layout":{"type":"constrained"}} -->
 <main class="wp-block-query">
+	<!-- wp:heading {"level"1} -->
+	<h1 class="wp-block-heading">Watch, Read, Listen</h1>
```

Comment:
We need an H1 on the index, so I added the text that is found above the blog in the home template.
But this text is not translation-ready. Do we really want to add a new pattern only for a heading..?

Reply:
there's already [a pattern](https://github.com/WordPress/twentytwentyfour/pull/86) for this

Reply:
Will update this PR once that is merged

Reply:
done!